essyncer-saas/.env.sample

```
@@ -1,2 +1,13 @@
+# MongoDB Configuration
 MONGO_URL=mongodb://mongo/erxes
-ELASTICSEARCH_URL=http://elasticsearch:9200
\ No newline at end of file
+CORE_MONGO_URL=mongodb://mongo/erxes_core
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORE_MONGO_URL key should go before the MONGO_URL key

(UnorderedKey)